redshift_connector/cursor.py

```diff
@@ -239,9 +244,31 @@ def executemany(self: "Cursor", operation, param_sets) -> "Cursor":
         self._row_count = -1 if -1 in rowcounts else sum(rowcounts)
         return self
 
+    def insert_data_bulk(self, filename, url, name, links, url_index, name_index):
+        """
+
+           runs a single bulk insert statement into the database.
+
+        """
+
+        sql_query = f"INSERT INTO  {links} ({url}, {name}) VALUES %s"
+
+        with open(filename) as csvfile:
+            reader = csv.reader(csvfile)
+            header = next(reader)
+            values_list = []
+
+            for row in reader:
+                values = (row[url_index], row[name_index])
+                values_list.append(values)
+
+            self.execute(sql_query, values_list)
+
     def callproc(self, procname, parameters=None):
         args = [] if parameters is None else parameters
-        operation = "CALL " + self.__sanitize_str(procname) + "(" + ", ".join(["%s" for _ in args]) + ")"
+        operation = "CALL " + \
```

Comment:
@Yash621 is this necessary? adds noise to the PR.

Reply:
fixed in revisions